Dockerfile

```diff
@@ -24,6 +24,7 @@ RUN set -eux \
     && grep -q '^gpgcheck=1' /etc/yum.repos.d/corretto.repo \
     && yum install -y java-1.8.0-amazon-corretto-devel-$version \
     && yum install -y fontconfig \
+    && yum update -y \
     && yum clean all
 
-ENV JAVA_HOME=/usr/lib/jvm/java-1.8.0-amazon-corretto
```

Comment:
is it intentional to remove the newline at the end ?

Reply:
Technically speaking, we always need to a newline to the end of all Linux files.
My editor (VIM) inserted it.   yes, it's intentional.

Reply:
sorry, I thought the newline was removed. but you are right and LGTM